op-deployer/pkg/deployer/integration_test/cli/manage_add_game_type_v2_test.go

```diff
@@ -106,17 +107,15 @@ func TestManageAddGameTypeV2_Integration(t *testing.T) {
 	workDir := runner.GetWorkDir()
 
 	// Test values - using arbitrary addresses for testing
-	l1ProxyAdminOwner := common.HexToAddress("0x1Eb2fFc903729a0F03966B917003800b145F56E2")
-	systemConfigProxy := common.HexToAddress("0x034edD2A225f7f429A63E0f1D2084B9E0A93b538")
+	l1ProxyAdminOwner := deployer.DefaultL1ProxyAdminOwnerSepolia
+	systemConfigProxy := deployer.DefaultSystemConfigProxySepolia
 
 	// Get OPCM V2 address from standard (using Sepolia chain ID for address lookup)
-	opcmV2, err := standard.OPCMImplAddressFor(11155111, standard.ContractsV500Tag)
+	opcmV2, err := standard.OPCMImplAddressFor(deployer.SepoliaChainID, standard.ContractsV500Tag)
 	require.NoError(t, err)
 
-	bytes32Type, err := abi.NewType("bytes32", "", nil)
-	require.NoError(t, err)
-	addressType, err := abi.NewType("address", "", nil)
-	require.NoError(t, err)
+	bytes32Type := deployer.Bytes32Type
```

Comment:
you could use [`mustType`](https://github.com/ethereum-optimism/optimism/blob/1c9d0fc93b9ed9f7b62f52e8ad69bb8278716c9c/op-deployer/pkg/deployer/opcm/contract.go#L101) instead:
```suggestion
    bytes32Type := opcm.MustType("bytes32")
    addressType := opcm.MustType("address")
```